gno.land/pkg/gnoweb/public/js/copy.js

```diff
@@ -1 +1 @@
+var s=class i{DOM;static FEEDBACK_DELAY=750;btnClicked=null;btnClickedIcons=[];isAnimationRunning=!1;static SELECTORS={button:".js-copy-btn",icon:"[data-copy-icon] > use",content:t=>`[data-copy-content="${t}"]`};constructor(){this.DOM={el:document.querySelector("main")},this.DOM.el?this.init():console.warn("Copy: Main container not found.")}init(){this.bindEvents()}bindEvents(){this.DOM.el?.addEventListener("click",this.handleClick.bind(this))}handleClick(t){let e=t.target.closest(i.SELECTORS.button);if(!e)return;this.btnClicked=e,this.btnClickedIcons=Array.from(e.querySelectorAll(i.SELECTORS.icon));let o=e.getAttribute("data-copy-txt"),r=e.getAttribute("data-copy-target");if(o){this.copyToClipboard(o,this.btnClickedIcons);return}if(r){let a=this.DOM.el?.querySelector(i.SELECTORS.content(r));if(!a){console.warn(`Copy: No content found for source "${r}".`);return}this.copyToClipboard(a,this.btnClickedIcons);return}console.warn("Copy: No content to copy found on the button.")}sanitizeContent(t){if(typeof t=="string")return t.trim();let n=t.innerHTML.replace(/<span[^>]*class="chroma-ln"[^>]*>[\s\S]*?<\/span>/g,""),e=document.createElement("div");return e.innerHTML=n,e.textContent?.trim()||""}toggleIcons(t){t.forEach(n=>{n.classList.toggle("hidden")})}showFeedback(t){!this.btnClicked||this.isAnimationRunning===!0||(this.isAnimationRunning=!0,this.toggleIcons(t),window.setTimeout(()=>{this.toggleIcons(t),this.isAnimationRunning=!1},i.FEEDBACK_DELAY))}async copyToClipboard(t,n){let e=this.sanitizeContent(t);if(!navigator.clipboard){console.error("Copy: Clipboard API is not supported in this browser."),this.showFeedback(n);return}try{await navigator.clipboard.writeText(e),this.showFeedback(n)}catch(o){console.error("Copy: Error while copying text.",o),this.showFeedback(n)}}},c=()=>new s;export{c as default};
```

Comment:
I don't know how you generate those files, but we should create them using the Makefile or npm script and retain the readable source. This PR has conflicts in this file, and resolving conflicts in such files is very challenging.

Reply:
It's actually the case. They are generated from makefile and pushed with the source